lib/ABI/FunctionType.cpp

```diff
@@ -559,6 +623,14 @@ class ConversionHelper {
       }
     }
 
+    // if (DA.Registers.size() > 1 && !AT::ArgumentsArePositionBased) {
+    //   // Register based data can be further affected by the endianness
+    //   // TODO: investigate the cases where this little "reverse" is not
+    //   enough. constexpr auto Architecture = model::ABI::getArchitecture(ABI);
+    //   if (model::Architecture::isLittleEndian(Architecture))
+    //     std::reverse(DA.Registers.begin(), DA.Registers.end());
+    // }
+
```

Comment:
We do not commit comment code

Reply:
Done: https://github.com/revng/revng/pull/148/commits/5e467c713d5d3465942fd48d20b0e0f5b9255c13.